src/vs/code/electron-main/auth.ts

```diff
@@ -76,6 +77,7 @@ export class ProxyAuthHandler extends Disposable {
 
 		const win = new BrowserWindow(opts);
 		const url = require.toUrl('vs/code/electron-sandbox/proxy/auth.html');
```

Comment:
I find this reads a bit weird. Isn't `url` that we get from `require.toUrl` actually the `fileUrl` and the thing we convert it to is the `vscodeFileUrl`?

All in all I wonder if this deserves a utility method somewhere:

```ts
toCodeFileUri(module) => URI.parse(require.toUrl('module')).with({ scheme: Schemas.vscodeFileResource, authority: 'localhost' }).toString()
```